### PR TITLE
chore(security): upgrade mendersoftware/go-lib-micro to the latest

### PR DESCRIPTION
**Pull request by tranchitella**
Ticket: MEN-5473
Changelog: title

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,17 +4,15 @@ go 1.14
 
 require (
 	github.com/gin-gonic/gin v1.7.7
-	github.com/mendersoftware/go-lib-micro v0.0.0-20200529072844-ecc6d0b89fa6
-	github.com/nats-io/nats-server/v2 v2.6.3 // indirect
-	github.com/nats-io/nats.go v1.13.1-0.20211018182449-f2416a8b1483
-	github.com/pelletier/go-toml v1.7.0 // indirect
+	github.com/mendersoftware/go-lib-micro v0.0.0-20220304124258-d8414885f1c3
+	github.com/nats-io/nats-server/v2 v2.7.3 // indirect
+	github.com/nats-io/nats.go v1.13.1-0.20220121202836-972a071d373d
```
[the remark this answers is not in the document]
Comment (tranchitella) [reply go.mod]: @alfrunes what about now?

**Review by alfrunes — APPROVED** [review]

File: go.mod (hunk 1 above)
[the remark this answers is not in the document]
Comment (tranchitella) [reply go.mod]: Maybe, but let's do it as a follow-up. I want to focus on security stuff now.

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
We can ignore the failure in `test_os_ent_migration`

**Pull request merged**